documents/building/README.md

```diff
@@ -20,14 +20,12 @@ Apache Celix aims to be support a broad range of UNIX platforms.
  
 Currently the [continuous integration build server](https://travis-ci.org/apache/celix) builds and tests Apache Celix for:
 
-* Ubuntu Trusty Tahr (14.04)
+* Ubuntu Trusty Tahr (18.04)
```

Comment:
The distribution name should be updated as well. Ubuntu 18.04 == Bionic Beaver.

http://releases.ubuntu.com/18.04.4/

Reply:
updated